README.md

```diff
@@ -291,6 +291,13 @@ Neither CUDA Runtime nor Driver APIs guarantees that GPU memory allocation
 functions return aligned addresses. Users are responsible for proper alignment
 of addresses passed to the library.
 
+Normally, pinning the same GPU address multiple times consumes the same amount
+of BAR1 space as doing it for the first time on that address because of the BAR1
+address reuse. However, GPU driver has a bug that prevents address reuse when
+pinning larger than a certain size (depending on platforms and GPU types). The
```

Comment:
I would be more generic, as in "some GPU drivers have a bug that ... "
That is because you never know on which branches the fix will be backported.

Reply:
Agreed. Fixed the paragraph.